server/controllers/meeting.js

```diff
@@ -0,0 +1,20 @@
+const Meeting = require("../models/Meeting");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+exports.createMeeting = asyncHandler(async (req, res, next) => {
+  const { username, duration } = req.body;
+  const loggedInUser = await User.findOne({ username }).then((doc) => {
+    const meeting = Meeting.create({
+      user_id: doc._id,
+      duration: duration,
+    });
+  });
+  res.status(200).json({ username, duration });
+});
+
+exports.getMeetings = asyncHandler(async (req, res, next) => {
+  Meeting.find({}).then((data) => {
+    res.json(data);
+  });
```

Comment:
This is not a good practice, this will allow a user on their browser to view every single meeting in the database. Even if it's not their meeting.